src/Workspaces/Remote/ServiceHub/ServiceHub.csproj

```diff
@@ -16,6 +16,7 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\CodeAnalysis.csproj" />
+    <ProjectReference Include="..\..\..\EditorFeatures\Core.Wpf\EditorFeatures.Wpf.csproj" />
```

Comment:
I think @KirillOsenkov tried to use service hub in VS4Mac as well. I probably need to take a look how to remove editor dependency completely from service hub.

Reply:
Yes, we're trying to integrate servicehub too!

Reply:
@Therzok We'll need to chat then as I suspect we would pretty easily put non-portable code into this assembly without realizing it.